DataFormats/METReco/src/classes_def.xml

```diff
@@ -169,9 +169,16 @@
   </class>
   <class name="edm::Wrapper<reco::HcalHaloData>"/>
 
+  <class name="HaloTowerStrip" ClassVersion="2">
+   <version ClassVersion="2" checksum="665188928"/>
+  </class>
+
+  <class name="std::vector<HaloTowerStrip>"/>
+  <class name="edm::Wrapper<std::vector<HaloTowerStrip> >"/>
+
   <class name="reco::CSCHaloData" ClassVersion="11">
```

Comment:
I'm a bit confused. Can you clarify what you mean by move around? I added the declaration of the HaloTowerStrip itself before the vector<>.